src/main/scala/centaur/api/CromwellClient.scala

```diff
@@ -11,19 +12,22 @@ import centaur.test.metadata.WorkflowMetadata
 import centaur.test.workflow.Workflow
 import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
 import akka.util.ByteString
-
 import centaur.api.CromwellStatusJsonSupport._
 import centaur.api.CromwellBackendsJsonSupport._
 import centaur.{CentaurConfig, SubmittedWorkflow, WorkflowStatus}
-
 import spray.json._
+
 import scala.collection.immutable
-import scala.concurrent.{Await, Future}
+import scala.concurrent.{Await, ExecutionContext, Future}
 import scala.concurrent.duration.FiniteDuration
-import scala.concurrent.ExecutionContext.Implicits.global
 import scala.util.Try
 
 object CromwellClient {
+  // Do not use scala.concurrent.ExecutionContext.Implicits.global as long as this is using Await.result
+  // See https://github.com/akka/akka-http/issues/602
+  // And https://github.com/viktorklang/blog/blob/master/Futures-in-Scala-2.12-part-7.md
```

Comment:
For some reason, seeing "part-7" just reminded me of [this](http://www.imdb.com/title/tt0093405/):

![mv5bmtyzmdg1ntk2nf5bml5banbnxkftztcwndqwndgymq _v1_uy268_cr2 0 182 268_al_](https://cloud.githubusercontent.com/assets/791985/24520956/49bb71a6-1558-11e7-922f-30abe7cb51d2.jpg)